jax/_src/numpy/ufuncs.py

```diff
@@ -872,14 +872,76 @@ def maximum(x: ArrayLike, y: ArrayLike, /) -> Array:
   """
   return lax.max(*promote_args("maximum", x, y))
 
-@implements(np.float_power, module='numpy')
+
 @partial(jit, inline=True)
 def float_power(x: ArrayLike, y: ArrayLike, /) -> Array:
+  """Calculate element-wise base ``x`` exponential of ``y``.
+
+  JAX implementation of :obj:`numpy.float_power`.
+
+  Args:
+    x: scalar or array. Specifies the bases.
+    y: scalar or array. Specifies the exponents. Both ``x`` and ``y`` should
```

Comment:
This reads strangely to me. Maybe remove the word "Both"?
```suggestion
    y: scalar or array. Specifies the exponents. ``x`` and ``y`` should
```

Same comment in the docstring below.

Reply:
Done